mixins/framework.js

```diff
@@ -1,17 +1,23 @@
+const fs = require('fs');
 const path = require('path');
-const glob = require('glob');
+const fastGlob = require('fast-glob');
 const helper = require('../config-helper');
+const packageRoot = require('../package-root');
 const EnactFrameworkPlugin = require('../plugins/dll/EnactFrameworkPlugin');
 
 module.exports = {
 	apply: function (config, opts = {}) {
-		const app = helper.appRoot();
+		const app = packageRoot();
+
+		// Scan for any polyfills.js entry file for potential re-usage.
+		const polyfillFile = helper.polyfillFile(config);
+
 		// Form list of framework entries; Every @enact/* js file as well as react/react-dom
 		config.entry = {
-			enact: glob
-				.sync('@enact/**/*.@(js|jsx|es6)', {
-					cwd: path.resolve(path.join(app, 'node_modules')),
-					nodir: true,
+			enact: fastGlob
+				.sync(['@enact/**/*.@(js|jsx|es6)'], {
```

Comment:
Seems a string is also accepted here.  Is there a reason to switch to array?  We do it in some places below and not in others.

Reply:
I was under the impression fastglob.sync only took arrays

Reply:
Good thing it doesn't. :D